asab/zookeeper/builder.py

```diff
@@ -0,0 +1,19 @@
+from urllib.parse import urlparse
+import aiozk
+
+"""
+This module builds ZooKeeper clients from Configs and urls
+"""
+
+async def build_client(Config, z_url):
+    # Parse URL
```

Comment:
It would be great if you document all three supported forms of ZooKeeper url.

Also, are all three supported here?

Reply:
it supports all 3 URLs , 
Absolute URLs are parsed without issues. But when Relative URLs are parsed 'netloc ' is empty in that case we read the entry for the servers from the [asab:zookeeper] section.

Reply:
it would require a lot of changes from several modules to use build__url.

Reply:
1) I don't understand the point of "it would require a lot of changes from several modules to..." ... it seems to be as a normal refactorization

2) I don't see form a code how it supports "Relative url with relative path" variant, it is not dealing with  `Config["asab:zookeeper"]["path"]` in any way.

Reply:
1. Got it!
2. Yes, we do not handle it in our code. I thought I will detail how the URL really works. If you think it can cause confusion I can get rid of it.

Reply:
2. - well, it seems to be quite straightforward to add a support for "Relative url with relative path" ...